### PR TITLE
Skip a hook based on the GITHOOKS_SKIP / _DISABLE env vars

### LINKED ISSUE #6: Environment variable to hard-skip specific hooks
Per a discussion with @timbunce, we should have a way to skip a hook. An environment variable GITHOOKS_SKIP that takes a comma-separated list of hooks to skip sounds desirable.

Additionally, in the case of the pre-commit hook being skipped this way, we should add a note in the commit message that this path was used (for future reference / review).

### PR DESCRIPTION
**Pull request by choroba**
Fixes #6.

### CODE REVIEW

File: lib/App/GitHooks.pm
Comment (guillaumeaubert) [inline lib/App/GitHooks.pm]: We may want to print a warning on STDERR here, to indicate that hooks were intentionally skipped. This will be a good reminder in case someone "inadvertently" adds this to their `.bash_profile`.

File: t/24-githooks-skip.t
Comment (guillaumeaubert) [inline t/24-githooks-skip.t]: I ❤️ tests. Thank you!

**Review by guillaumeaubert — CHANGES_REQUESTED** [review]

File: lib/App/GitHooks.pm
Comment (choroba) [inline lib/App/GitHooks.pm]: Yes. We should also create a follow-up ticket to add a note to the commit message if the pre-commit hook was skipped.

**Comment by choroba** [comment]
OK, here's a fixup. Let me know if it's OK, I'll squash the commits then.

**Comment by guillaumeaubert** [comment]
Looks great - thank you for the tweaks and extra tests! I'll merge after you squash the commits then.

Are you planning to do other PRs? Just checking to see if I should hold off a release on CPAN.

**Comment by choroba** [comment]
Squashed. As February's over, I'm moving to a next task :-) Thanks for the help.

**Comment by guillaumeaubert** [comment]
Thank you so much again, @choroba! Merged, and I'll do a release shortly with your changes.

**Pull request merged**
